Yoshi.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Yoshi"
-  s.version          = "1.1.0"
+  s.version          = "2.0"
```

Comment:
it's convention, take a look here https://github.com/Alamofire/AlamofireImage/commit/33c66155f7990890c730b5847909e12fbae93842#diff-3c97d208da64ea8b4acd0306e8caf4adL3